docker/yarn/setup.sh

```diff
@@ -1,8 +1,7 @@
 #!/bin/bash
 
-userdel node
+# workaround for macos default groupid (20)
+groupmod -g 24 dialout
```

Comment:
maybe just change to 4 digit number instead of trying to pinpoint in between numbers (or call `groupadd` with `-o` option).